### DISCUSSION TITLE
WHIP 2: WiHi DAO

### DISCUSSION THREAD

**Discussion by chiumichael** [wihi-labs/WHIP#13] [Ideas]
- Authors: [Mark Ballandies](https://twitter.com/BallandiesMC)
- Start Date: 2023-09-28
- Category: Governance
- Tracking Issue: #3
----
## Summary

The WiHi DAO is the organizational structure that the WiHi community gives itself to achieve its vision of facilitating weather predictions everywhere on earth anytime. It follows the principles of collective intelligence and self-organization to manage its complex 3-sided market consisting of weather data providers, data consumers and platform maintainers/ developers/ facilitators. 

Link to WHIP:

https://github.com/wihi-labs/WHIP/blob/main/0002-wihi-dao/main.md

**Comment by urosvk** [comment]
ack
